azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -1380,6 +1429,43 @@ WithManagedCreate withNewDataDiskFromImage(int imageLun,
         }
 
         /**
+         * The stage of the virtual machine definition allowing to enable boot diagnostics.
+         */
+        @Beta(Beta.SinceVersion.V1_2_0)
```

Comment:
On the new withers, it'll work better if we put the `@Beta`s on the methods themselves as well. (more discoverable)